packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+
+:::
+
+## Organizing Using `Communication.topic`
+
+The `Communication.topic` element represents a description of the main focus or content of the message. It allows you to link the communication to a specific topic, or subject matter, making it easier for recipients to understand the purpose and content of the message.
+
+It is easy to think of the topic as if it is the subject line of an email for a given communication. In that sense, it is useful to use that level of specificity when defining a topic. For example, a topic could be an appointment for a given patient on a given date. This topic could be assigned to all communications related to that appointment.
+
+**Example: **
+
+```ts
+{
+  resourceType: 'Communication',
+  id: 'example-communication-2',
+  topic: {
+    text: 'In-person appointment with Homer Simpson on April 10th, 2023. Made by telephone.',
+    coding: [
+      {
+        code: '185420007',
+        system: 'http://snomed.info.sct',
+        display: 'In-person appointment made by telephone - Homer Simpson - 04-10-2023'
+      }
+    ]
+  }
+}
+```
+
+## Modeling Threads
+
+You can also use the `Communication` resource to model threading, enabling you to represent the relationship between multiple communications that are part of the same discussion. This is a way to organize related communications and logically link messages so that the context of the conversation is maintained.
+
+Threading can best be modeled using the `Communication.partOf` and `Communication.inResponseTo` elements. These properties allow you to reference other communications, establishing a relationship, or thread, between new and existing communications.
+
+## Threading with `Communication.partOf`
+
+The `partOf` element represents a larger resource of which the communication is a component or step. It can reference any resource type, allowing us to refer to other communications to create a thread. The `partOf` element is best used to create a thread in which each message is linked back to the original parent message, creating an ancestor-descendant relationship.
+
+When creating a thread the initial parent message will not have a value stored in `partOf`. For each response create a new communication resource, setting `partOf` to reference the original communication resource. As the thread grows, each communication should continue to point to that original parent, which will form a group of communications with a common reference point.
+
+In the example below, despite that this is a thread of communications in order, the initial message is the communication being referenced by both responses in the `partOf` property.
```

Comment:
Don't think this example is quite right. I wouldn't expect the parent Comm to have a payload